### PR TITLE
Access mapping - qradar defaults

### PR DESCRIPTION
**Pull request by meirwah**

### CODE REVIEW

File: Classifiers/classifier-QRadar.json [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+				}
+			},
+			"dontMapEventToLabels": false
+		}
+	},
+	"custom": true,
+	"transformer": {
+		"simple": "",
+		"complex": {
+			"root": "categories",
```
Comment (asafshen) [inline Classifiers/classifier-QRadar.json]: Why are we using complex instead of simple?

File: Classifiers/classifier-QRadar.json [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+			"accessor": "[0]",
+			"transformers": []
+		}
+	},
+	"keyTypeMap": {
+		"SIM User Authentication": "Access",
+		"User Login Failure": "Access"
+	},
+	"brandName": "QRadar",
+	"releaseNotes": "-"
```
Comment (asafshen) [inline Classifiers/classifier-QRadar.json]: lets try to add release notes that explain a bit about the mapping/classificaiton
`QRadar classification and mapping for 'Access' incident type`

**Comment by asafshen** [comment]
@meirwah can we merge this?

**Comment by meirwah** [comment]
Let's wait with this.. I want to gather feedback on the splunk mapping only first...

**Pull request merged**